docs/apis-tools/operate-api/overview.md

```diff
@@ -18,6 +18,10 @@ A detailed API description is also available as Swagger UI at `https://${base-ur
 
 For SaaS: `https://${REGION}.operate.camunda.io/${CLUSTER_ID}/swagger-ui.html`, and for Self-Managed installations: `http://localhost:8080/swagger-ui.html`.
 
+:::note
+Find your region and cluster id under connection information in your client credentials.
```

Comment:
Is there a bolded UI element we should be looking for here?

Reply:
Unfortunately, that's part of the problem. **Region** has to distinct values depending on where in the UI you find it. I've proposed changing this in the UI to "region id" - https://camunda.slack.com/archives/C0589TC88UU/p1689175806936099?thread_ts=1689000734.350649&cid=C0589TC88UU